lib/applications/field-resolvers.ts

```diff
@@ -0,0 +1,18 @@
+import { Resolver } from '@lib/resolvers'; // Resolver type
+import { Application } from '@lib/types';
+
+export const applicationApplicantResolver: Resolver<Application> = async (
+  parent,
+  _args,
+  { prisma }
+) => {
+  return await prisma.applicant.findUnique({ where: { id: parent?.applicantId || undefined } });
+};
+
+export const applicationPermitResolver: Resolver<Application> = async (
+  parent,
+  _args,
+  { prisma }
+) => {
+  return await prisma.permit.findMany({ where: { applicationId: parent?.id } });
```

Comment:
This should be `findUnique` since every application can only be related to at most one permit.

Reply:
The way our schema is defined, the `Application` model has multiple `Permit` objects associated with it. To fix this, we would have to modify our schema and change a bunch of types. For sake of getting an MVP out, I think it'll be fine to leave as is and fix later - `findMany` with the ID restriction will still work as desired, since each `Permit` is only associated with one object.

Reply:
Good point. Nice catch! I will add a ticket to the backlog to address this post-MVP.